public/statics/config/client.js

```
@@ -113,6 +113,129 @@ function showErrorMessage(message) {
+  setTimeout(() => {
+    successDiv.remove();
+  }, 3000);
+}
+
+// Convert value by type
+function convertValueByType(value, type) {
+  const trimmedValue = value.trim();
+  
+  switch (type) {
+    case 'boolean':
+      const lowerValue = trimmedValue.toLowerCase();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)